vnext/Shared/JSI/RuntimeHolder.h

```diff
@@ -6,6 +6,8 @@
 namespace facebook {
 namespace jsi {
 
+enum RuntimeType { Chakra = 0, Hermes = 1, V8 = 2, V8NAPI = 3 };
```

Comment:
>enum [](http://example.com/codeflow?start=0&length=4)

(nit) use enum class for better type safety

Reply:
Reusing an existing enum now ..